src/components/Editor/Footer.js

```diff
@@ -48,15 +48,18 @@ class SourceFooter extends PureComponent<Props> {
     }
 
     const tooltip = L10N.getStr("sourceTabs.prettyPrint");
-    const type = "prettyPrint";
+    const isPrettyPrinting =
+      selectedSource.get("prettyStatus") == "prettifying";
 
+    const type = "prettyPrint";
     return (
       <button
         onClick={() => togglePrettyPrint(selectedSource.get("id"))}
         className={classnames("action", type, {
-          active: sourceLoaded,
+          active: sourceLoaded || isPrettyPrinting,
```

Comment:
why do we need `|| isPrettyPrinting`

Reply:
I thought, when active class is added, button's ui changes somehow to represent that it's active, so that user can know that the action is in progress.
But atm, adding active doesn't do anything.

Reply:
Got it